sql/sqlserver/server.go

```diff
@@ -842,3 +865,28 @@ func unmarshalValue(col structured.ColumnDescriptor, kv client.KeyValue) sqlwire
 	}
 	return d
 }
+
+type valMap map[string]sqlwire.Datum
+
+func (m valMap) Get(name string) (parser.Datum, bool) {
```

Comment:
the `Env` interface should explain the second return value (not in diff so I'm commenting here).